src/mudlet-lua/lua/GUIUtils.lua

```diff
@@ -70,7 +70,7 @@ color_table = {
   cadet_blue                = { 95, 158, 160 },
   CadetBlue                 = { 95, 158, 160 },
   chartreuse                = { 127, 255, 0 },
-  chocolate                 = { 210, 105, 30 },
+  chocolate                 = { 210, 105, 30 },f
```

Comment:
:question: A typo, methinks...!

Reply:
Woopsadoodle! My bad haha, yeah that's not intended at all.

Reply:
That's weird, I don't know how I missed it...